doc/source/whatsnew/v0.23.0.txt

```diff
@@ -345,6 +345,7 @@ Other Enhancements
 - :meth:`DataFrame.to_sql` now performs a multivalue insert if the underlying connection supports itk rather than inserting row by row.
   ``SQLAlchemy`` dialects supporting multivalue inserts include: ``mysql``, ``postgresql``, ``sqlite`` and any dialect with ``supports_multivalues_insert``. (:issue:`14315`, :issue:`8953`)
 - :func:`read_html` now accepts a ``displayed_only`` keyword argument to controls whether or not hidden elements are parsed (``True`` by default) (:issue:`20027`)
+- :func:`DataFrame.to_parquet` now accepts a ``metadata`` keyword argument - the object passed updates key value file metadata generated by pandas. If pandas key is present - default pandas value is overwritten and warning is issued. Default ``None`` value means standard pandas metadata is used. (``None`` by default) (:issue:`20521`)
```

Comment:
This should be a short highlight with a link to the API docs or a new section in `io.rst` if necessary.

Reply:
Do you mean this should be shorter and sentence moved to API docs?

Reply:
Yeah, a link to the API documentation is probably best here.